css/css-flexbox/percentage-max-height-004.html

```diff
@@ -0,0 +1,29 @@
+<!doctype quirks>
+<title>CSS Flexbox Test: replaced element in nested flexboxes has correct height to resolve percentage against </title>
+<link rel="author" title="Ziran Sun" href="mailto:zsun@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox/#flex-containers">
+<link rel="help" href="https://drafts.csswg.org/css-sizing-3/#percentage-sizing">
+<link href="support/flexbox.css" rel="stylesheet">
+<link rel="match" href="percentage-max-height-004-ref.html">
+<meta name="assert" content="Replaced children with % max-height are sized correctly when their parents are flex items with percentage heights in quirks mode." />
+
+
+<style>
+.flex {
+  display:flex;
+  flex-direction:column;
+}
+.percent-height {
+  height: 100%;
+}
+.max-sizes {
+  max-height: 100%;
+  max-width: 100%;
+}
+</style>
+<div class="percent-height flex">
```

Comment:
We should add a text here describing the expected result, like:

"Test passes if there is a filled green square and no red"

Reply:
As explained earlier,  Inserting text in the test script introduces a scrolling bar and probably make it easy to leave it for now since it is really not the main focus of this test.